src/components/SearchableMultiSelect/SearchableMultiSelect.spec.jsx

```diff
@@ -233,6 +233,34 @@ describe('SearchableMultiSelect', () => {
 				expect(onSelect).toHaveBeenCalledWith([1]);
 			});
 		});
+
+		describe('Error', () => {
+			it('should apply the appropriate classNames to the saerch', () => {
+				const wrapper = shallow(
+					<SearchableMultiSelect Error={'Erroring out'}>
+						<Option>option a</Option>
+						<Option>option b</Option>
+					</SearchableMultiSelect>
+				);
+
+				const searchWrapper = wrapper.find('.lucid-SearchableMultiSelect-search-is-error');
+
+				expect(searchWrapper).toBeTruthy();
+			});
+
+			it('should render out the error div', () => {
```

Comment:
Maybe add a test case specifically for the `true` case that should render the input in an `-is-error` state but _not_ render the extra div

Reply:
updated/added test case with the revised search class logic